packages/core/src/app/customer/CheckoutButtonList.tsx

```diff
@@ -36,28 +36,32 @@ export const SUPPORTED_METHODS: string[] = [
 export interface CheckoutButtonListProps {
     methodIds?: string[];
     isInitializing?: boolean;
-    copywritingStringId?: string;
+    hideText?: boolean;
     checkEmbeddedSupport?(methodIds: string[]): void;
     deinitialize(options: CustomerRequestOptions): void;
     initialize(options: CustomerInitializeOptions): void;
     onError?(error: Error): void;
 }
 
+export const filterUnsupportedMethodIds = (methodIds:string[]):string[] => {
```

Comment:
Is our ESLint not working? IIRC I thought we have a rule on [key-spacing](https://eslint.org/docs/latest/rules/key-spacing)?

Reply:
Strange, this should have been picked up.